File: Programming/Python/00/0002 Min Max.py
# Input data lines
N = int(input())

# Input number and put them in the list
number_list = []
for i in range(0,N):
    num = int(input())
    number_list.append(num)

# Output min and max of the number list
print(min(number_list))
print(max(number_list))

File: Programming/C++/00/0002 Min Max.cpp
#include<bits/stdc++.h>
using namespace std;
int N, number[1005];
int maximum = -2000000001, minimum = 2000000001;
int main ()
{
    // Input the number of data lines
    cin >> N;

    // Input N numbers and put them in the array
    for(int i=1;i<=N;i++)
    {
        cin >> number[i];
        if(number[i] > maximum)
        {
            maximum = number[i];
        }
        if(number[i] < minimum)
        {
            minimum = number[i];
        }
    }

    // Output the minimum and maximum value
    cout << minimum << "\n";
    cout << maximum;

    // End of the program
    return 0;
}